lib/kino/control.ex

```diff
@@ -0,0 +1,126 @@
+defmodule Kino.Control do
+  @moduledoc """
+  Various widgets for user interactions.
+
+  Each widget is a UI control element that the user interacts
+  with, consequenty producing an event stream.
+
+  Those widgets are often useful paired with `Kino.Frame` for
+  presenting content that changes upon user interactions.
+
+  ## Examples
+
+  First, create a control and make sure it is rendered,
+  either by placing it at the end of a code cell or by
+  explicitly rendering it with `Kino.render/1`.
+
+      button = Kino.Control.button("Hello")
+
+  Next, to receive events from the control, a process needs to
+  subscribe to it and specify pick a name to distinguish the
+  events.
+
+      Kino.Control.subscribe(button, :hello)
```

Comment:
What if instead of giving a name, `subscribe` returns a reference. And you can use this reference to associate to a certain client/name?